library/Ivoz/Provider/Infrastructure/Persistence/Doctrine/Mapping/Recording.RecordingAbstract.orm.yml

```diff
@@ -30,7 +30,7 @@ Ivoz\Provider\Domain\Model\Recording\RecordingAbstract:
       precision: 10
       scale: 3
       options:
-        default: '0.000'
+        default: 0.000
```

Comment:
Float fields have 0.000 as default, while decimal fields have 0.0
Is this intended?

Also precision and scale may not affect float fields.

Reply:
It turns out that you can specify float precision `duration float(10,3) NOT NULL DEFAULT '0.000'`. I'll set 0.0 as default value